Sources/UIPanGestureRecognizer.swift

```diff
@@ -107,16 +114,10 @@ open class UIPanGestureRecognizer: UIGestureRecognizer {
         reset()
     }
 
-    open override func touchesCancelled(_ touches: Set<UITouch>, with event: UIEvent) {
-        super.touchesCancelled(touches, with: event)
-        reset()
-    }
-
     private func reset() {
         trackedTouch = nil
         previousTouchesMovedTimestamp = nil
         touchesMovedTimestamp = nil
         initialTouchPoint = .zero
-        state = .possible
```

Comment:
i'm a bit concerned about this change. why do we not also reset the state in `reset`? seems surprising to me as a reader